### PR TITLE
improve comments and style

### PR DESCRIPTION
**Pull request by axel-h**
collect some trivial things form the other PR that could be merged easily:
- trivial: improve style for code and comments
- add comment about empty Arch_finaliseInterrupt()
- add comment about handleUnknownSyscall()

### CODE REVIEW

File: src/arch/arm/kernel/boot.c [hunk 1]
```diff
@@ -538,16 +538,16 @@ static BOOT_CODE bool_t try_init_kernel(
...
     bi_finalise();
 
-    /* make everything written by the kernel visible to userland. Cleaning to PoC is not
-     * strictly neccessary, but performance is not critical here so clean and invalidate
-     * everything to PoC */
+    /* make everything written by the kernel visible to userland. Cleaning to
+     * PoC is not strictly neccessary, but performance is not critical here so
+     * clean and invalidate everything to PoC
+     */
     cleanInvalidateL1Caches();
```
Comment (lsf37) [inline src/arch/arm/kernel/boot.c:545]: Nothing to do with your PR, but it looks like that comment is actually wrong. `cleanInvalidateL1Caches()` doesn't clean to PoC. This was wrong already in the commit that introduced it in 2013. I think the code is right in this case: flushing L1 and invalidating the TLB is good enough here, unless we are handing any frames to user level uncached that we modified here, which I don't think we do.

The rest is handled in `retype`, which (now) does flush correctly.

**Review by lsf37 — APPROVED** [review]
LGTM. We can fix that comment in a separate PR if you like.

File: src/arch/arm/kernel/boot.c (hunk 1 above)
Comment (axel-h) [reply src/arch/arm/kernel/boot.c:545]: I'll extend the comment.

Comment (axel-h) [reply src/arch/arm/kernel/boot.c]: change the comment, see https://github.com/seL4/seL4/pull/517/commits/c75e6c76d09309b0d3db795f74ef050430743df1

**Review by lsf37 — APPROVED** [review]

**Pull request merged**